data/mods/gen1stadium0/conditions.ts

```diff
@@ -0,0 +1,99 @@
+// Stadium Zero's statuses are a mix of RBY and Stadium's. It uses Stadium sleep, but RBY burn/tox/par. It's hard to understand.
+export const Conditions: {[k: string]: ModdedConditionData} = {
+	brn: {
+		name: 'brn',
+		effectType: 'Status',
+		onStart(target) {
+			this.add('-status', target, 'brn');
+			target.addVolatile('brnattackdrop');// TESTME: This should be ignored after a boost.
```

Comment:
Do you want help writing tests for these?

Reply:
That could be handy, yeah!